app/views/coronavirus_landing_page/components/shared/_accordion_sections.html.erb

```diff
@@ -53,19 +51,21 @@
   border_top: border_top,
   font_size: "m"
 } if heading %>
-<div data-module="gem-track-click gtm-click-tracking">
+<div data-module="gtm-click-tracking">
   <div data-module="toggle-attribute">
     <%
       show_all_attributes = {
         event_name: "select_content",
         type: "accordion",
         index: 0,
         index_total: number_of_accordion_sections,
-        section: "n/a"
       }
     %>
     <%= render 'govuk_publishing_components/components/accordion', {
       heading_level: 3,
+      data_attributes: {
+        module: "gtm-track-click",
```

Comment:
The module should be `gtm-click-tracking` to work correctly, see: https://github.com/alphagov/govuk_publishing_components/blob/main/docs/analytics-gtm/gtm-click-tracking.md